static/app/components/segmentedControl.tsx

```diff
@@ -171,7 +171,7 @@ const SegmentWrap = styled('label')<{
   isDisabled?: boolean;
 }>`
   position: relative;
-  display: block;
+  display: flex;
```

Comment:
`LabelWrap` (nested inside `SegmentWrap`) doesn't span the full height of `SegmentWrap`. Making `SegmentWrap` as flex display stretches `LabelWrap` out to full height.